PharoObjectModel/PharoObjectModel.pier

```diff
@@ -219,46 +219,50 @@ by instances of that class (and of its subclasses).
-those that create instances of the class, like ==Color class>>blue== and those
-that perform a utility function, like ==Color class>>showColorCube==. This is
+examples. You will see that there are two kinds of methods defined on a class:
+''instance creation methods'', like ==Color class>>blue==, and those that perform
+a utility function, like ==Dictionary class>>removeUnreferencedKeys==. This is
 typical, although you will occasionally find class methods used in other ways.
 
 It is convenient to place utility methods on the class side because they can be
 executed without having to create any additional objects first. Indeed, many of
 them will contain a comment designed to make it easy to execute them.
 
-Browse method ==Color class>>>showColorCube==, double-click just inside the
-quotes on the comment =="Color showColorCube"== and type CMD\-D. You will see
-the effect of executing this method. Select World > restore display (r) to undo
-the effects.
-
-For those familiar with Java and C\+\+,  class methods may seem similar to
-static methods. However, the uniformity of Pharo means that they are somewhat
-different: whereas Java static methods are really just statically-resolved
-procedures, Smalltalk class methods are dynamically-dispatched methods. This
-means that inheritance, overriding and super-sends work for class methods in
-Smalltalk, whereas they don't work for static methods in Java.
+% Commenting out for the moment: showColorCube is no longer in the image.
+% TODO: create a different example
+%Browse method ==Color class>>showColorCube==, double-click just inside the
+%quotes on the comment =="Color showColorCube"== and press ==CMD-d==. You will
+%see the effect of executing this method. Select
+%==World > System > Restore display (r)== to undo the effects.
+
+For those familiar with Java and C\+\+, class methods may seem similar to static
+methods. However, the uniformity of the Pharo object model (where classes are
+just regular objects) means that they are somewhat different: whereas Java
+static methods are really just statically-resolved procedures, Pharo class
+methods are dynamically-dispatched methods. This means that inheritance,
+overriding and super-sends work for class methods in Pharo, whereas they don't
+work for static methods in Java.
 
 !!!Class instance variables
 
 With ordinary instance variables, all the instances of a class have the same set
-of variable names, and the instances of its subclasses inherit those names;
-however, each instance has its own private set of values.The story is exactly
-the same with class instance variables: each class has its own private class
-instance variables.A subclass will inherit those class instance variables, ''but
-it has its own private copies of those variables''. Just as objects don't share
-instance variables, neither do classes and their subclasses share class instance
-variables.
+of variable names (though each instance has its own private set of values), and
```

Comment:
Is there a way we can rephrase it so it's clearer? 
The previous wording, "and the instances of its subclasses inherit those names", is still there (it's right on line 250), I just added the parenthetical clarification to make it clear that the classes don't just inherit the names, but the inherited variables are separate/private.